katalog/thanos/thanos-components/thanos-compact/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - thanos-compact.yaml
+  - servicemonitor.yml
+
+images:
+  - name: thanos
+    newName: quay.io/thanos/thanos
```

Comment:
Should we clone the image to our quay? I can't remember what has been decided on that topic.

Reply:
mmm i dunno, but i never saw any cloned image on other projects so far. If is intended to be done, i guess that we have to organize something that is out of topic with this PR @ralgozino

Reply:
Right now, we continue from upstream. We have to think about building our custom images, but.... currently, out of scope.